docs/source/api/touch.rst

```diff
@@ -0,0 +1,126 @@
+#####
+TOUCH
+#####
+
+About
+-----
+
+Touch sensor is a peripheral, that can be used to sense electrical changes on respective GPIO pins. Therefore these touch sensors 
+are also known as capacitive sensors. For example, if you touch any of these pins, the touch sensor will sense output according to 
+electrical charge on your finger. These pins can be easily integrated into capacitive pads, and replace mechanical buttons.
+
```

Comment:
The touch sensor will count the number of charge/discharge cycles over a fixed period of time. A finger electrical charge will change this number, by changing the RC circuit attached to the touch sensor. 
The change of this count will be used to validate if a touch has happened or not.

Reply:
Do you mean to add this to the section or replace some part?

Reply:
I think we can replace a part of the text because the sensor doesn't exactly sense electrical changes, but it has an internal oscilator circuit that measures charge/discharge frequency. When touched the capaitance of the RC circuit changes, therefore the time for charging/discharging.  The TouchRead() will return the number of cycles (charges/discharges) in a certain time (meas).

Reply:
Is this ok? (if yes, I will commit the change :))

Touch sensor is a peripheral, that has an internal oscilator circuit that measures charge/discharge frequency over a fixed period of time on respective GPIO pins. 
Therefore these touch sensors are also known as capacitive sensors. For example, if you touch any of these pins, finger electrical charge will change this number, 
by changing the RC circuit attached to the touch sensor. The touchRead() will return the number of cycles (charges/discharges) in a certain time (meas). 
The change of this count will be used to validate if a touch has happened or not.
These pins can be easily integrated into capacitive pads, and replace mechanical buttons.

Reply:
Touch sensor is a peripheral, that has an internal oscilator circuit **and it** measures charge/discharge frequency over a fixed period of time on respective GPIO pins.

Reply:
finger electrical charge will change this **number of cycles**

Reply:
I think it is all...

Reply:
Done.